### ISSUE TITLE
Commitlint: commit body lines must not be longer than 100 characters

### ISSUE THREAD

**Issue by MikeDombo** [dependabot/dependabot-core#5923]
### Is there an existing issue for this?
#2445

- [X] I have searched the existing issues

### Package ecosystem

maven

### What you expected to see, versus what you actually saw

Some dependabot commits cannot pass commitlint in CI due to the body's lines being over 100 characters.
Ex: https://github.com/aws-greengrass/aws-greengrass-logging-java/pull/131

```
Error: You have commit messages with errors

⧗   input: chore(deps): bump maven-shade-plugin from 3.2.2 to 3.4.0

Bumps [maven-shade-plugin](https://github.com/apache/maven-shade-plugin) from 3.2.2 to 3.4.0.
- [Release notes](https://github.com/apache/maven-shade-plugin/releases)
- [Commits](https://github.com/apache/maven-shade-plugin/compare/maven-shade-plugin-3.2.2...maven-shade-plugin-3.4.0)
✖   body's lines must not be longer than 100 characters [body-max-line-length]
```

**Comment by MikeDombo** [comment]
https://github.com/dependabot/dependabot-core/issues/2445
This issue was closed in favor of #1666, but the PR which fixed #1666 does not fix this problem.

**Comment by jeffwidman** [comment]
Thanks, you're right, so I re-opened #2445 as it has more context... and will close this as a dupe of the original. Thanks again for the pointer.
